src/models/Event.js

```diff
@@ -0,0 +1,53 @@
+const Group = require("./Group");
```

Comment:
Is this branch dependent on [Marice's](48-group-data-model-and-group-endpoints)? Should I merge her branch in to get this running?